### PR TITLE
[DOCTEST] Fix the documentation of RoCBert

### PR DESCRIPTION
**Pull request by ArthurZucker**
# What does this PR do?
Fixes the documetnation test

### CODE REVIEW

File: src/transformers/models/roc_bert/modeling_roc_bert.py [hunk 1]
```diff
@@ -52,6 +52,22 @@
...
 
+# Base model docstring
+_EXPECTED_OUTPUT_SHAPE = [1, 8, 768]
+
+# Token Classification output
+_TOK_CLASS_EXPECTED_OUTPUT = ['LABEL_0', 'LABEL_1', 'LABEL_0', 'LABEL_0', 'LABEL_0', 'LABEL_1', 'LABEL_1', 'LABEL_0', 'LABEL_1', 'LABEL_1', 'LABEL_1', 'LABEL_1', 'LABEL_1', 'LABEL_1']
+
+# SequenceClassification docstring
+_SEQ_CLASS_EXPECTED_LOSS = 0.0
+_SEQ_CLASS_EXPECTED_OUTPUT = "'LABEL_1'"
```
Comment (sgugger) [inline src/transformers/models/roc_bert/modeling_roc_bert.py:63]: Don't we have any model with labels? Adding those examples with abstract label just does not make any sense...

Comment (sgugger) [reply src/transformers/models/roc_bert/modeling_roc_bert.py:63]: Would be better no?

**Comment by ArthurZucker** [comment]
Sorry for the late fix, tests are passing locally.

**Review by sgugger — APPROVED** [review]
Way better, thanks!

**Pull request merged**